src/kontrol/__main__.py

```diff
@@ -683,6 +687,20 @@ def _parse_test_version_tuple(value: str) -> tuple[str, int | None]:
         type=str,
         help='Path to write summary .sol files, relative to foundry root',
     )
+    summary_args.add_argument(
+        '--comment-generated-files',
+        dest='comment_generated_file',
+        default='// This file was autogenerated by running `kontrol summary`. Do not edit this file manually.\n',
+        type=str,
+        help='Comment to write at the top of the auto generated summary files',
+    )
+    summary_args.add_argument(
+        '--license',
+        dest='license',
+        default=None,
```

Comment:
Here you could have the `license` to default on `UNLICENSED`.
```suggestion
        default='UNLICENSED',
```